modular_skyrat/modules/customization/modules/clothing/under/misc.dm

```diff
@@ -687,3 +687,25 @@
 	icon_state = "christmasfemaleg"
 
 
+/obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/helltaker
```

Comment:
Something other than helltaker

Reply:
It's in code no one will ever really see. It doesn't really matter.

Reply:
I'd rather it was something else.

Reply:
yeah, we already have something with the ...data...thingy called helltaker

Reply:
what about....

hellraiser :^)